packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -108,22 +106,26 @@ cbp-list {
 }
 
   //Link List
+  &[variant="link"]{
+    padding-inline-start: var(--cbp-space-8x);
+    
+    li{
+      list-style-type: none;
+  
+      .linkListIcon, 
+      .linkListDescriptionIcon{
+        margin-inline-start: calc(var(--cbp-space-4x) * -1);
+      }
+  
+      .linkListIcon{
+        color: var(--cbp-link-list-icon-color);
+      }
 
-  &[variant="link"] li{
-    list-style-type: none;
-
-    .linkListIcon{
-      color: var(--cbp-link-list-icon-color);
-    }          
-  }
-
-  &[variant="link-external"]{
-
-    .linkListDescription{
-      color: var(--cbp-link-list-description-color);
-      font-style: italic;
-      font-size: var(--cbp-font-size-body);
+      .linkListDescription{
```

Comment:
This is bothering me for the same reason as above. Maybe link lists should be handled by a component instead? Too much structure is expected without providing that structure.

In any case, you shouldn't use mixed case in CSS classes. convert to dashed or BEM and we can discuss/revisit later.